examples/filetimes.py

```diff
@@ -90,7 +90,7 @@ def benchmark(fn, args, filetype=None):
         if p.dftype == 'dask' and DD_FORCE_LOAD:
             if DEBUG:
                 print("DEBUG: Force-loading Dask dataframe", flush=True)
-            df = res.persist()
+            res = res.persist()
```

Comment:
After looking into things a bit more, I found that `persist()` does not act by side-effect only. It a makes a copy of the graph: https://github.com/dask/dask/blob/master/dask/base.py#L457-L460 , modifies the copy, and returns it.